Makefile

```diff
@@ -0,0 +1,157 @@
+mkfile_dir := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+##### Variables used
+# (can be overriden using command line args to make)
+# Paths used in build.
+BUILD_DIR:=$(abspath ${mkfile_dir}/../build)
+GECKO_SOURCE_DIR=$(abspath ${mkfile_dir}/../gecko-dev)
+GAIA_SOURCE_DIR=$(abspath ${mkfile_dir}/../gaia)
+# Used to increment .deb package version
+DATETIME=$(shell date +%d%m%y%H%M)
+GECKO_VERSION=46
+CUSTOM_MOZCONFIG?=
+RESOLUTION=800x600
+B2G_PROFILE_PATH=${HOME}/.mozilla/b2g/xsession.profile
+######
+
+##### Git repo validity check
+# gecko/gaia must have the Ronin root commit
+GECKO_RONIN_COMMIT=4b1114685cac9b758e923071eff5bf4350400ebd
+GAIA_RONIN_COMMIT=64cf0c8cf0cf2bf15f6fef5a8e6a1eeb40e18c95
+# Nice attempt but doesn't work sadly...
+git_has_commit = $(shell cd $(1) && git merge-base --is-ancestor $(2) HEAD && exit 0 || exit 1)
```

Comment:
Normally, you shouldn't need `exit 0` and `exit 1`, as git merge-base already returns those. But maybe that's not the most important thing here. What was your problem exactly?